roles/core/repositories_client/tasks/main.yml

```diff
@@ -5,6 +5,6 @@
     - internal
 
 - name: include_tasks ░ Use OS dedicated task
-  include_tasks: "{{ (ansible_facts.distribution|lower|replace(' ','_')) }}_{{ ansible_facts.distribution_major_version }}.yml"
+  include_tasks: "{{ (ansible_facts.distribution|replace(' ','_')) }}_{{ ansible_facts.distribution_major_version }}.yml"
```

Comment:
Can you use `with_first_found` to include `ansible_facts.os_family` and use RedHat.yml instead?

This would avoid duplication below:
```
e632367079c7390982a9b039367968e26eda9f2d52c01dd452d35ccb4826bbaa  roles/core/repositories_client/tasks/CloudLinux_8.yml
e632367079c7390982a9b039367968e26eda9f2d52c01dd452d35ccb4826bbaa  roles/core/repositories_client/tasks/OracleLinux_8.yml
e632367079c7390982a9b039367968e26eda9f2d52c01dd452d35ccb4826bbaa  roles/core/repositories_client/tasks/Redhat_8.yml
e632367079c7390982a9b039367968e26eda9f2d52c01dd452d35ccb4826bbaa  roles/core/repositories_client/tasks/RedHat_9.yml
```